gitlab/plugin.go

```diff
@@ -16,43 +16,45 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"gitlab_epic":                       tableEpic(),
+			"gitlab_group_access_request":       tableGroupAccessRequest(),
+			"gitlab_group_hook":                 tableGroupHook(),
 			"gitlab_group_iteration":            tableGroupIteration(),
-			"gitlab_project_iteration":          tableProjectIteration(),
+			"gitlab_group_member":               tableGroupMember(),
+			"gitlab_group_project":              tableGroupProject(),
 			"gitlab_group_push_rule":            tableGroupPushRule(),
-			"gitlab_group_hook":                 tableGroupHook(),
-			"gitlab_project_protected_branch":   tableProjectProtectedBranch(),
-			"gitlab_project_pages_domain":       tableProjectPagesDomain(),
-			"gitlab_setting":                    tableSetting(),
-			"gitlab_merge_request_change":       tableMergeRequestChange(),
-			"gitlab_project_variable":           tableProjectVariable(),
+			"gitlab_group_subgroup":             tableGroupSubgroup(),
+			"gitlab_group":                      tableGroup(),
 			"gitlab_group_variable":             tableGroupVariable(),
 			"gitlab_instance_variable":          tableInstanceVariable(),
-			"gitlab_group_access_request":       tableGroupAccessRequest(),
+			"gitlab_issue":                      tableIssue(),
+			"gitlab_merge_request_change":       tableMergeRequestChange(),
+			"gitlab_merge_request":              tableMergeRequest(),
+			"gitlab_my_event":                  tableMyEvents(),
+			"gitlab_my_issue":                   tableMyIssue(),
+			"gitlab_my_project":                 tableMyProject(),
 			"gitlab_project_access_request":     tableProjectAccessRequest(),
-			"gitlab_application":                tableApplication(),
 			"gitlab_project_container_registry": tableProjectContainerRegistry(),
 			"gitlab_project_deployment":         tableProjectDeployment(),
-			"gitlab_group_project":              tableGroupProject(),
-			"gitlab_group_subgroup":             tableGroupSubgroup(),
+			"gitlab_project_iteration":          tableProjectIteration(),
+			"gitlab_project_job":                tableProjectJob(),
+			"gitlab_project_member":             tableProjectMember(),
+			"gitlab_project_pages_domain":       tableProjectPagesDomain(),
+			"gitlab_project_pipeline_detail":    tableProjectPipelineDetail(),
+			"gitlab_project_pipeline":           tableProjectPipeline(),
+			"gitlab_project_protected_branch":   tableProjectProtectedBranch(),
+			"gitlab_project_repository_file":    tableProjectRepositoryFile(),
+			"gitlab_project_repository":         tableProjectRepository(),
+			"gitlab_project":                    tableProject(),
+			"gitlab_project_variable":           tableProjectVariable(),
+			"gitlab_setting":                    tableSetting(),
+			"gitlab_snippet":                    tableSnippet(),
+			"gitlab_user_event":                tableUserEvents(),
+			"gitlab_user":                       tableUser(),
+			"gitlab_version":                    tableVersion(),
```

Comment:
I believe you've attempted to sort this, albeit there's some weirdness to the sort i.e `gitlab_group` & `gitlab_project` seemingly are out of order, suggested alphabetical sort order:
```
			"gitlab_application":                tableApplication(),
			"gitlab_branch":                     tableBranch(),
			"gitlab_commit":                     tableCommit(),
			"gitlab_epic":                       tableEpic(),
			"gitlab_group":                      tableGroup(),
			"gitlab_group_access_request":       tableGroupAccessRequest(),
			"gitlab_group_hook":                 tableGroupHook(),
			"gitlab_group_iteration":            tableGroupIteration(),
			"gitlab_group_member":               tableGroupMember(),
			"gitlab_group_project":              tableGroupProject(),
			"gitlab_group_push_rule":            tableGroupPushRule(),
			"gitlab_group_subgroup":             tableGroupSubgroup(),
			"gitlab_group_variable":             tableGroupVariable(),
			"gitlab_instance_variable":          tableInstanceVariable(),
			"gitlab_issue":                      tableIssue(),
			"gitlab_merge_request":              tableMergeRequest(),
			"gitlab_merge_request_change":       tableMergeRequestChange(),
			"gitlab_my_event":                   tableMyEvents(),
			"gitlab_my_issue":                   tableMyIssue(),
			"gitlab_my_project":                 tableMyProject(),
			"gitlab_project":                    tableProject(),
			"gitlab_project_access_request":     tableProjectAccessRequest(),
			"gitlab_project_container_registry": tableProjectContainerRegistry(),
			"gitlab_project_deployment":         tableProjectDeployment(),
			"gitlab_project_iteration":          tableProjectIteration(),
			"gitlab_project_job":                tableProjectJob(),
			"gitlab_project_member":             tableProjectMember(),
			"gitlab_project_pages_domain":       tableProjectPagesDomain(),
			"gitlab_project_pipeline":           tableProjectPipeline(),
			"gitlab_project_pipeline_detail":    tableProjectPipelineDetail(),
			"gitlab_project_protected_branch":   tableProjectProtectedBranch(),
			"gitlab_project_repository":         tableProjectRepository(),
			"gitlab_project_repository_file":    tableProjectRepositoryFile(),
			"gitlab_project_variable":           tableProjectVariable(),
			"gitlab_setting":                    tableSetting(),
			"gitlab_snippet":                    tableSnippet(),
			"gitlab_user":                       tableUser(),
			"gitlab_user_event":                 tableUserEvents(),
			"gitlab_version":                    tableVersion(),
```

Reply:
I just did `!}sort` in vim. I just do that in lists like this to make it easier to find things. Sadly `"` comes after `_`...  I'll fix that.

Reply:
Fixed in 52e3e11